### PR TITLE
PAI submitter for TF and XGB evaluation

### PR DESCRIPTION
**Pull request by lhw362950217**
Realated to #2656

### CODE REVIEW

**Review by shendiaomo — DISMISSED** [review]

File: python/runtime/pai/submitter.py [hunk 1]
```diff
@@ -814,4 +814,98 @@ def submit_pai_explain(datasource, select, result_table, model_name,
...
+    metrics = []
+    if model_type == EstimatorType.XGBOOST:
+        metrics.append("accuracy_score")
+    elif model_type == EstimatorType.TENSORFLOW:
+        metrics.append("Accuracy")
+
+    met_conf = model_attrs.get("validation.metrics") or model_attrs.get(
+        "validationMetrics")
+    if met_conf:
+        [
```
Comment (typhoonzero) [inline python/runtime/pai/submitter.py:834]: Clear default metrics before appending?

File: python/runtime/pai/submitter.py [hunk 2]
```diff
@@ -814,4 +814,98 @@ def submit_pai_explain(datasource, select, result_table, model_name,
...
+
+    Args:
+        datasource: current datasource
+        result_table: the table name to save result
+        metrics: array of evaluation metrics names
+    """
+    drop_tables([result_table], datasource)
+    # Always add loss
+    ext_metrics = ["loss"]
+    ext_metrics.extend(metrics)
```
Comment (typhoonzero) [inline python/runtime/pai/submitter.py]: Check `metrics` is of type `list` before running `extend`?

File: python/runtime/pai/submitter.py [hunk 3]
```diff
@@ -814,4 +814,102 @@ def submit_pai_explain(datasource, select, result_table, model_name,
...
+
+    Args:
+        datasource: current datasource
+        result_table: the table name to save result
+        metrics: list of evaluation metrics names
+    """
+    drop_tables([result_table], datasource)
+    # Always add loss
+    ext_metrics = ["loss"]
+    if isinstance(metrics, list):
```
Comment (typhoonzero) [inline python/runtime/pai/submitter.py:860]: else raise `ValueError`?

**Review by typhoonzero — APPROVED** [review]
Can merge after the tiny fix

File: python/runtime/pai/submitter.py (hunk 3 above)
Comment (lhw362950217) [reply python/runtime/pai/submitter.py:860]: I will fix this in following PR.

**Pull request merged**